recipes/rdkit/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "rdkit" %}
+{% set version = "Release_2017_09" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rdkit/rdkit/archive/{{ version }}.tar.gz
+  sha256: ab83bb4d3c5986d22da0ce28d352e5f911830b63f18fee85537cebfa3df28e82
+  patches:
+    - rdpaths.patch
+    - rdconfig.patch  # [win]
+
+build:
+  skip: true  # [win and py3k]
+  skip: true  # [osx and (py27 or py35)]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - requests
+    - boost <=1.64
+    - python
+    - numpy 1.11.*
+    - pillow
+    - freetype
+    - eigen
+    - pandas
+    - cairo
+    - pkg-config  # [not win]
+  run:
+    - cairo
+    - boost <=1.64
+    - python
+    - six
+    - numpy >=1.11
+
+test:
+  imports:
+    - rdkit
+    - rdkit.Avalon
+    - rdkit.Chem
+    - rdkit.Chem.AllChem
+    - rdkit.DataManip
+    - rdkit.Dbase
+    - rdkit.DistanceGeometry
+    - rdkit.ForceField
+    - rdkit.Geometry
+    - rdkit.ML
+    - rdkit.Numerics
+    - rdkit.SimDivFilters
+    - rdkit.VLib
+    - rdkit.VLib.NodeLib
+
+about:
+  home: http://rdkit.org
+  license: BSD
```

Comment:
Did we get a version for this (e.g. `BSD 3-Clause` if it is 3-Clause)? Also is there a license file included in the source somewhere?

Reply:
it's BSD 3-Clause
license file is in github: https://github.com/rdkit/rdkit/blob/master/license.txt

Reply:
Thanks. So would update `license` on this line and add `license_file: license.txt` on the following line.

Reply:
Do we need to actually include the license file in the recipe directory? or just point to the file in the source repo?

Reply:
You can point to the file in source. `license_file: license.txt` would look at the file `license.txt` in the root folder of the source.